packages/bot/src/struct/Command.ts

```diff
@@ -22,12 +28,37 @@ export type CommandBody<Type extends ApplicationCommandType> = RESTPostAPIApplic
 };
 
 export interface Command<Type extends ApplicationCommandType = ApplicationCommandType> {
+	readonly containsSubcommands: false;
```

Comment:
I would make this `readonly containsSubcommands?: false;` - leave it optional so we don't have to specify it everywhere. it being `true` on Subcommands is enough